packages/legend-query-builder/src/components/fetch-structure/QueryBuilderTDSPanel.tsx

```diff
@@ -644,6 +648,14 @@ const QueryBuilderProjectionColumnEditor = observer(
       [handleDrop],
     );
 
+    const lastColumnBeingDragged =
```

Comment:
TODO: I like the informative nature of these, but we can probably make it more succinct. I also start to wonder how we could potentially componentize this kind of interaction

Also, come to think about it, we don't need to put states in `QueryBuilderTDSState`, we probably can make do by putting those states using `useState` in the parent/list component